references/proxy-template/pipeline.sh

```diff
@@ -14,21 +14,81 @@
 # limitations under the License.
 
 set -e
-set -x
 
 SCRIPTPATH="$( cd "$(dirname "$0")" || exit >/dev/null 2>&1 ; pwd -P )"
 
+if [ -z "$1" ] || [ "$1" = "--apigeeapi" ];then
+    apiversion="--apigeeapi"
+    TEST_HOST="$APIGEE_ORG-$APIGEE_ENV.apigee.net"
+fi
+
+if [ -z "$1" ] || [ "$1" = "--googleapi" ];then
+    apiversion="--googleapi"
+    TEST_HOST="$APIGEE_X_HOSTNAME"
```

Comment:
if $1 is not set then you'll still only run tests against one deployment option. It probably makes sense to duplicate the logic below more or create a deploy.sh file that you can use for both.

Reply:
ok done Daniel